prac_05/hex_colours.py

```diff
@@ -0,0 +1,14 @@
+# Hex Colours
+
+
+COLOUR_AND_CODE = {"Aliceblue": "#0048ba", "Antiquewhite": "#faebd7", "Aqua": "#00ffff", "Bistre": "#3d2b1f",
+                   "Black": "#000000", "Carmine": "#960018", "Fulvous": "#e48400", "Goldenrod1": "ffc125",
+                   "Indigo": "4b0082", "Magenta3": "cd00cd"}
+
+colour_name = input("Enter a colour name: ").title()
+while colour_name != "":
+    print(f"The code for {colour_name} is {COLOUR_AND_CODE.get(colour_name)}")
+    colour_name = input("Enter a colour name: ").title()
+
```

Comment:
There would be an error if the user don't enter a color that is in the set. Also you could just use dictionary instead of set.